ibis/pyspark/compiler.py

```diff
@@ -71,8 +71,8 @@ def compile_selection(t, expr, scope, **kwargs):
     # Cache compile results for tables
     op = expr.op()
 
-    # TODO: Support predicates and sort_keys
-    if op.predicates or op.sort_keys:
+    # TODO: Support sort_keys
```

Comment:
could you please file an issue and link it here? i worry we lose track of TODOs if we step away for too long

Reply:
Yep, could you link me to the JIRA project?

Reply:
i think just an issue here in Github is fine

Reply:
https://github.com/ibis-project/ibis/issues/1957